tests/components/tuya/fixtures/msp_mzuro9jgjs7uiryn.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
Please add file to `DEVICE_MOCKS` in `__init__.py`
https://github.com/home-assistant/core/blob/d5bd93ebda2ba9fc83e93a0a22fa1493e7218811/tests/components/tuya/__init__.py#L15-L17

Please also ensure that it is formatted correctly by `prettier` linter